plugins/gestures.koplugin/main.lua

```diff
@@ -1195,13 +1195,22 @@ function Gestures:multiswipeAction(multiswipe_directions, ges)
     end
 end
 
-function Gestures:onIgnoreHoldCorners(ignore_hold_corners)
+function Gestures:onIgnoreHoldCorners(ignore_hold_corners, no_notification)
     if ignore_hold_corners == nil then
         G_reader_settings:flipNilOrFalse("ignore_hold_corners")
     else
         G_reader_settings:saveSetting("ignore_hold_corners", ignore_hold_corners)
     end
     self.ignore_hold_corners = G_reader_settings:isTrue("ignore_hold_corners")
+
+    if no_notification then return true end -- when toggled from menu
```

Comment:
I'd leave out the presumption that just so happens to be true within this PR.
```suggestion
    if no_notification then return true end
```